### PR TITLE
9530 android swrve logger

### PR DESCRIPTION
**Pull request by shanetmoore**
New toggleable SwrveLogger class which is a wrapper around the Android Logger. It allows users to disable logs through the SwrveConfig class (this has been requested once already)

### CODE REVIEW

**Comment by shanetmoore** [comment]
@Sergio-Mira @dominicmarmionswrve

File: SwrveSDK/src/main/java/com/swrve/sdk/SwrveLogger.java [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+import android.util.Log;
+
+import com.swrve.sdk.SwrveLogger;
+
+public class SwrveLogger {
+
+    private static final String LOG_TAG = "SWRVE";
+
+    // Config/Active/Inactive
+    private static boolean isActive = true;
```
Comment (dominicmarmionswrve) [inline SwrveSDK/src/main/java/com/swrve/sdk/SwrveLogger.java]: Would BuildConfig.debug be a better default?
private static boolean isActive = BuildConfig.debug;

**Comment by dominicmarmionswrve** [comment]
Could we also replace all the occurrences of printStackTrace with this new Logger?

**Comment by shanetmoore** [comment]
Done and Done

**Comment by shanetmoore** [comment]
The stack trace loggers might not actually be needed in some cases.

I'm not sure printing the stacktrace provides over logging them with Log.e. IE `Log.e(tag, message, exception)` vs `e.printStackTrace()`

**Comment by dominicmarmionswrve** [comment]
:rocket:

**Pull request merged**